integrations/huggingface-transformers/tutorials/bert_sparse_transfer_learning.md

```diff
@@ -0,0 +1,149 @@
+| Binary Classification      | [QQP](https://quoradata.quora.com/First-Quora-Dataset-Release-Question-Pairs) | `python transformers/examples/pytorch/text-classification/run_glue.py --model_name_or_path bert-base-uncased --task_name qqp --do_train --do_eval --evaluation_strategy epoch --per_device_train_batch_size 32 --learning_rate 5e-5 --max_seq_length 128 --output_dir models/teacher --num_train_epochs 2 --seed 2021`                                 |
+| Multi-Class Classification | [MNLI](https://cims.nyu.edu/~sbowman/multinli/)                               | `python transformers/examples/pytorch/text-classification/run_glue.py --model_name_or_path bert-base-uncased --task_name mnli --do_train --do_eval --evaluation_strategy epoch --per_device_train_batch_size 32 --learning_rate 5e-5 --max_seq_length 128 --output_dir models/teacher --num_train_epochs 2 --seed 2021`                                |
+| Named Entity Recognition   | [CoNNL2003](https://www.clips.uantwerpen.be/conll2003/ner/)                   | `python transformers/examples/pytorch/token-classification/run_ner.py --model_name_or_path bert-base-uncased --dataset_name conll2003 --do_train --do_eval --evaluation_strategy epoch --per_device_train_batch_size 32 --learning_rate 5e-5 --output_dir models/teacher --preprocessing_num_workers 16 --num_train_epochs 5 --seed 2021`              |
+| Sentiment Analysis         | [SST2](https://nlp.stanford.edu/sentiment/)                                   | `python transformers/examples/pytorch/text-classification/run_glue.py --model_name_or_path bert-base-uncased --task_name sst2 --do_train --do_eval --evaluation_strategy epoch --per_device_train_batch_size 32 --learning_rate 5e-5 --max_seq_length 128 --output_dir models/teacher --preprocessing_num_workers 16 --num_train_epochs 2 --seed 2021` |
+
+Select the training command that applies to your use case and then run it in your training environment.
+The time to execute the training commands will differ according to dataset and training environment, but generally, they should run to completion in less than 12 hours.
+Once the command has completed, you will have a deployable sparse model located in `models/teacher`.
+
+You are ready to transfer learn the model.
+
+## Transfer Learning the Model
+
+With the dense teacher now trained to convergence, you will begin the sparse transfer learning with distillation with a recipe.
+The dense teacher will distill knowledge into the sparse architecture, therefore increasing its performance while ideally converging to the dense solution's accuracy.
+The recipe encodes the hyperparameters necessary for transfer learning the sparse architecture.
+Specifically, it ensures that the sparsity is preserved through the training process.
```

Comment:
Somewhere here maybe we should add some sentences about the ConstantPruningModifier because it's the only extra piece needed for transferring a sparse model. It also shows that we make sparse transfer learning easy to do (just add these 3 lines in your regular training recipe etc).

Reply:
I think this is diving into a bit too much detail for a user at this level. We likely should have a separate one that explores what a general sparse transfer recipe is composed of